packages/riipen-ui-docs/src/modules/components/MarkdownPage.jsx

```diff
@@ -11,141 +18,174 @@ import Menu from "src/modules/components/Menu";
 import TOC from "src/modules/components/TOC";
 import { getSections } from "src/utils/parseMarkdown";
 
-import AppBar from "riipen-ui/components/AppBar";
-import Divider from "riipen-ui/components/Divider";
-import ThemeContext from "riipen-ui/styles/ThemeContext";
+const MarkdownPage = ({ req, path, reqSource, title }) => {
+  const theme = React.useContext(ThemeContext);
+
+  const [anchorState, setAnchorState] = React.useState(null);
+  const [menuOpen, setMenuOpen] = React.useState(false);
 
-class MarkdownPage extends React.Component {
-  static propTypes = {
-    path: PropTypes.string.isRequired,
-    req: PropTypes.func.isRequired,
-    reqSource: PropTypes.func,
-    title: PropTypes.string
+  const handleMenuOpen = (event) => {
+    document.body.style.overflow = menuOpen ? "auto" : "hidden";
```

Comment:
Force lock scroll.
I think popover might be supposed to do this but it looks like it only works on mount. @bolinkd can you confirm if that's desired behaviour to only lock on mount (and not when anchorEl/lockScroll props change)? You are the Popover guy in my mind :P 
I think Popover can use a Convert to Function facelift anyways :)

Reply:
I believe the original intention is that the scroll of the window behind is disabled so that the scrolling the user does is captured by the popover (it might be the case where we want that interaction configurable)